changelog/2023-11-22.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: 2023 Major Release
+---
+
+import Image from '../components/image'
+
+<Image
+  src='/img/changelog/202311/regional.png'
+  alt='Smoother regional analysis view'
+/>
+
+We have been hard at work on Conveyal's major release for 2023. This release includes **v7.0** of the R5 routing engine and...
```

Comment:
Will this wording draw more attention to the fact that there are not many releases in 2023? Maybe we should say something like "Conveyal's late 2023 major release."

Reply:
I agree, I suggest "November, 2023" as that would fit the style that we've used in the most recent changelog entries.